master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: Etcdless Hosted Install
+title: Kubernetes Datastore Driver
 ---
 
 This document describes installing Calico on Kubernetes in a mode that does not require access to an etcd cluster.  
 This mode uses the Kubernetes API as the datastore.  Note that this feature
 currently comes with a number of limitations, namely:
 
-- it supports policy enforcement only and does not yet support Calico BGP networking.
```

Comment:
This should now read "It supports BGP full-mesh networking, but does not yet support BGP peer configuration"